storage/replica.go

```diff
@@ -132,8 +133,10 @@ func usesTimestampCache(r roachpb.Request) bool {
 // committed to the Raft log, the command is executed and the result returned
 // via the done channel.
 type pendingCmd struct {
-	ctx  context.Context
-	done chan roachpb.ResponseWithError // Used to signal waiting RPC handler
+	ctx     context.Context
+	idKey   cmdIDKey
```

Comment:
idKey can be removed if change signature of proposePendingCmdLocked to take idKey as input parameter.